packages/node/src/private-keys-generator.ts

```diff
@@ -0,0 +1,154 @@
+import { Node } from "@counterfactual/types";
+import { Wallet } from "ethers";
+import { BigNumber } from "ethers/utils";
+import { fromExtendedKey, fromMnemonic } from "ethers/utils/hdnode";
+import log from "loglevel";
+import { Memoize } from "typescript-memoize";
+
+import { CF_PATH } from "./constants";
+
+export const EXTENDED_PRIVATE_KEY_PATH = "EXTENDED_PRIVATE_KEY";
+
+export class PrivateKeysGenerator {
```

Comment:
`implements IPrivateKeyGenerator`?

Reply:
This class itself doesn't implement the interface, it's just a wrapper invoking the function that implements the interface and does the needed error checking for whether the generated private key is valid.

Reply:
I'll rename the class, this is incorrectly named.